tests/product/test_create_images.py

```diff
@@ -0,0 +1,98 @@
+from nose.plugins.attrib import attr
+from subprocess import check_output
+from sys import argv
+from tests.no_hadoop_bare_image_provider import NoHadoopBareImageProvider
+from tests.product.base_product_case import BaseProductTestCase
+
+import unittest
+
+
+@attr('quarantine')
+class TestSetupImages(BaseProductTestCase):
```

Comment:
i took the functions out of the test case and put them in the module. you can see if that's what you had in mind.